app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb

```diff
@@ -0,0 +1,121 @@
+module ForemanVirtWhoConfigure
+  module Api
+    module V2
+      class ConfigsController < ::Api::V2::BaseController
+        include ::Api::Version2
+        # include Api::TaxonomyScope
+        include ForemanVirtWhoConfigure::Concerns::ConfigParameters if Foreman::Version.new.short >= '1.13'
+        resource_description do
+          api_base_url "/foreman_virt_who_configure/api"
+        end
+
+        before_action :find_resource, :only => %w{show deploy_script update destroy}
+
+        api :GET, '/configs', N_("List of virt-who configurations")
+        api :GET, "/organizations/:organization_id/configs", N_("List of virt-who configurations per organization")
+        param_group :search_and_pagination, ::Api::V2::BaseController
+
+        def index
+          @configs = resource_scope_for_index
+        end
+
+        api :GET, "/configs/:id/", N_("Show a virt-who configuration")
+        param :id, :identifier, :required => true
+
+        def show
+        end
+
+        api :GET, "/configs/:id/deploy_script/", N_("Renders a deploy script for the specified virt-who configuration")
+        param :id, :identifier, :required => true
+        def deploy_script
+          respond_to do |format|
+            format.text { render :text => @config.virt_who_bash_script }
+            format.sh { render :text => @config.virt_who_bash_script }
+            format.json
+          end
+        end
+
+        def_param_group :config do
+          param :foreman_virt_who_configure_config, Hash, :required => true, :action_aware => true do
+            param :name, String, :desc => N_("Configuration name"), :required => true
+            param :interval, Config::AVAILABLE_INTERVALS.keys.map(&:to_i), :desc => N_("Configuration interval in minutes"), :required => true
+            param :filtering_mode, Config::FILTERING_MODES.keys.map(&:to_i), :desc => N_("Hypervisor filtering mode, %{unlimited} means no filtering, %{whitelist} means whitelist, %{blacklist} means blacklist") % { :unlimited => Config::UNLIMITED, :whitelist => Config::WHITELIST, :blacklist => Config::BLACKLIST }, :required => false
+            param :whitelist, String, :desc => N_("Hypervisor whitelist, applicable only when filtering mode is set to 1"), :required => false
+            param :blacklist, String, :desc => N_("Hypervisor blacklist, applicable only when filtering mode is set to 2"), :required => false
```

Comment:
We should also mention what is the format: `"Wildcards and regular expressions are supported, multiple records must be separated by comma."`
The same for `:whitelist`